Ports/libksba/package.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=libksba
+version=1.5.1
+useconfigure=true
+#configopts="--with-libgpg-error-prefix=${SERENITY_BUILD_DIR}/Root/usr/local"
```

Comment:
What is the significance of this commented-out part?

Reply:
It's gone now. I just copied the package.sh file from one of the other ports and forgot to remove the comments.